content/scout/exceptions.md

```diff
@@ -0,0 +1,164 @@
+refer to the [`vexctl` GitHub repository](https://github.com/openvex/vexctl).
+
+For an introduction to VEX, you may also want to check out this use-case guide:
+[Suppress image vulnerabilities with VEX](./guides/vex.md).
+
+## Attach exceptions to images
+
+When you've created an exception,
+you can attach it to your image in the following ways:
+
+- Embed the document in the [image filesystem](#image-filesystem)
+- Attach the document as an [attestation](#attestation)
+
+### Image filesystem
+
+Embedding exceptions directly on the image filesystem is a good option if you
+know the exceptions ahead of time, before you build the image. And it's easy;
+just `COPY` the VEX document to the image in your Dockerfile.
+
+The downside with this approach is that you can't change or update the
+exception later. Image layers are immutable, so anything you put in the image's
+filesystem is there forever. Attaching the document as an
+[attestation](#attestation) provides better flexibility.
+
+To embed a VEX document on the image filesystem, `COPY` the file into the image
+as part of the image build. The following example shows how to copy all VEX
+documents under `.vex/` in the build context, to `/var/lib/db` in the image.
+
+```dockerfile
+# syntax=docker/dockerfile:1
+
+FROM alpine
+COPY .vex/* /var/lib/db/
+```
+
+The filename of the VEX document must match the `*.vex.json` glob pattern.
+It doesn't matter where on the image's filesystem you store the file.
+
+Note that the copied files must be part of the filesystem of the final image,
+For multi-stage builds, the documents must persist in the final stage.
+
+### Attestation
+
+To attach exceptions as an attestation, you can use the `docker scout
+attestation add` CLI command.
+
+Using attestations is the recommended option for attaching exceptions to
+images. You can attach attestations to the image after it's built and pushed to
+a registry, meaning you don't need to create the VEX documents before you
+build. Additionally, having the exceptions attached to the image as
+attestations means consumers can inspect the exceptions for an image, directly
+from the registry.
+
+To attach an attestation to an image:
+
+1. Build the image and push it to a registry.
+
+   ```console
+   $ docker build --provenance=true --sbom=true --tag <IMAGE> --push .
+   ```
+
+2. Attach the exception to the image as an attestation.
+
+   ```console
+   $ docker scout attestation add \
+     --file <cve-id>.vex.json \
+     --predicate-type https://openvex.dev/ns/v0.2.0 \
+     <IMAGE>
+   ```
+
+## View exceptions
+
+The **Exceptions** page on the [Docker Scout Dashboard](https://scout.docker.com/)
+lists the exceptions for all images in your organization. Selecting a row in
+the list opens the exception side panel, which displays more information about
+the exception and where it comes from.
+
+To view all exceptions for a specific image tag:
+
+{{< tabs >}}
+{{< tab name="Dashboard" >}}
```

Comment:
**suggestion:** Connected to [my previous comment](https://github.com/docker/docs/pull/19700/files#r1547370090), if we do remove step one maybe it's good if we clarify what "Dashboard" we are referring to.
```suggestion
{{< tab name="Docker Scout Dashboard" >}}
```

Reply:
Maybe this is a docs wide change rather than a fix just here?

Reply:
Good callout. I think when I first started using these we only had the scout dashboard, now there's a few (dbc, admin, maybe more coming) let me update to be clear.